src/chapters/typescript/arrays.rst

```diff
@@ -0,0 +1,75 @@
+Arrays in TypeScript
+====================
+
+.. index:: ! tuple
+
+Arrays in TypeScript must contain values of the same type. 
+When declaring an array, the type needs to be declared.
+
+.. sourcecode:: js
+   :linenos:
+
+   let arrayName = number [];
+
+   arrayName = [10,9,8];
+
+What if the array needs to hold values of different types?
+
+Now, we need a **tuple**. A tuple is a special structure in TypeScript that can hold as many values as needed of different types.
+
+.. sourcecode:: js
+   :linenos:
+
+   let tupleName = [number, string, number];
+
+   tupleName = [10, "9", 8];
+
+Examples
+--------
+
+In JavaScript, we would declare an array holding items in our cargo hold like so:
+
+.. sourcecode:: js
+
+   let cargoHold = ['oxygen tanks', 'space suits', 'parrot', 'instruction manual', 'meal packs', 'slinky', 'security blanket'];
+
+In TypeScript, we would declare the same ``cargoHold`` array a little differently:
+
+.. sourcecode:: js
+
+   let cargoHold = string [];
+   cargoHold = ['oxygen tanks', 'space suits', 'parrot', 'instruction manual', 'meal packs', 'slinky', 'security blanket'];
```

Comment:
I really like this example. However I think the ts syntax is wrong. I got an error when I ran that code. How about this code? `let cargoHold: string[] = ['oxygen tanks', 'space suits', 'parrot', 'instruction manual', 'meal packs', 'slinky', 'security blanket'];`

Reply:
Fixed!